pkg/splunk/controller/util.go

```diff
@@ -211,15 +211,30 @@ func MergePodSpecUpdates(ctx context.Context, current *corev1.PodSpec, revised *
 				result = true
 			}
 
-			if current.Containers[idx].LivenessProbe != nil && revised.Containers[idx].LivenessProbe != nil &&
-				current.Containers[idx].LivenessProbe.InitialDelaySeconds != revised.Containers[idx].LivenessProbe.InitialDelaySeconds {
-				current.Containers[idx].LivenessProbe.InitialDelaySeconds = revised.Containers[idx].LivenessProbe.InitialDelaySeconds
+			/*
```

Comment:
Nit: Comments not needed? Apologies if you already were planning to remove it.

Reply:
Thanks for reminder. Will update.